pipelines/manual-release-pipeline.yml

```diff
@@ -22,6 +22,7 @@ groups:
   - "Regional Counts"
   - "Report Deployment Success"
   - "Trigger Terraform"
+  - "Preview Terraform Changes"
```

Comment:
Should be in the `Infrastructure` group also

Reply:
Good spot...done